test/alternator/test_metrics.py

```diff
@@ -130,25 +131,50 @@ def test_batch_write_item(test_table_s, metrics):
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchWriteItem', 'le': '86.000000'}], [2, {'op': 'BatchWriteItem', 'le': '103.000000'}]]):
+        test_table_s.meta.client.batch_write_item(RequestItems = {
+            test_table_s.name: items})
+        test_table_s.meta.client.batch_write_item(RequestItems = {
+            test_table_s.name: items2})
+
+def test_batch_get_item_count(test_table_s, metrics):
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchWriteItem', 'le': '2.000000'}], [1, {'op': 'BatchGetItem', 'le': '3.000000'}]]):
+        test_table_s.meta.client.batch_get_item(RequestItems = {
+            test_table_s.name: {'Keys': [{'p': random_string()}, {'p': random_string()}, {'p': random_string()}], 'ConsistentRead': True}})
+    keys = [{'p': random_string()} for i in range(100) ]
+    keys2 = [{'p': random_string()} for i in range(100) ]
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchGetItem', 'le': '86.000000'}], [2, {'op': 'BatchGetItem', 'le': '103.000000'}]]):
+        test_table_s.meta.client.batch_get_item(RequestItems = {
+            test_table_s.name: {'Keys': keys, 'ConsistentRead': True}})
+        test_table_s.meta.client.batch_get_item(RequestItems = {
+            test_table_s.name: {'Keys': keys2, 'ConsistentRead': True}})
+
+
 # Reproduces issue #9406:
 def test_batch_get_item(test_table_s, metrics):
     with check_increases_operation(metrics, ['BatchGetItem']):
         test_table_s.meta.client.batch_get_item(RequestItems = {
             test_table_s.name: {'Keys': [{'p': random_string()}], 'ConsistentRead': True}})
 
-def test_batch_write_item_count(test_table_s, metrics):
+def test_batch_write_item_count_total(test_table_s, metrics):
     with check_increases_operation(metrics, ['BatchWriteItem'], metric_name='scylla_alternator_batch_item_count', expected_value=2):
         test_table_s.meta.client.batch_write_item(RequestItems = {
             test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}, {'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}]})
 
-def test_batch_get_item_count(test_table_s, metrics):
+def test_batch_get_item_count_total(test_table_s, metrics):
     with check_increases_operation(metrics, ['BatchGetItem'], metric_name='scylla_alternator_batch_item_count', expected_value=2):
         test_table_s.meta.client.batch_get_item(RequestItems = {
             test_table_s.name: {'Keys': [{'p': random_string()}, {'p': random_string()}], 'ConsistentRead': True}})
 
 KB = 1024
 def test_rcu(test_table_s, metrics):
-    with check_increases_metric_exact(metrics, 'scylla_alternator_rcu_total', 4):
+    with check_increases_metric_exact(metrics, 'scylla_alternator_rcu_total', [[4, None]]):
```

Comment:
This change in the function's signature is a bit uglier than I anticipated, but I think since it's just in one place, it's fine.

Reply:
Unfortunately, I found out I needed to test for multiple values, so the trivial one more optional parameter was sufficient. I came to the same conclusion as you did, as it's just one place, it's ugly but not worth a dedicated function